lib/OverlayUIComponents/orange_button.dart

```diff
@@ -33,6 +33,9 @@ class EditorJSOrangeButton extends StatelessWidget {
       case ('externaldeeplink'):
         buttonIcon = Icon(Icons.arrow_outward);
         break;
+      case ('integration'):
+        buttonIcon = Icon(Icons.cable_rounded);
```

Comment:
probably ok for now but i think integrations should have an icon matching their integration or be selectable by the user or the api service implementation